packages/ui/src/working-groups/hooks/useOpenings.ts

```diff
@@ -42,8 +42,11 @@ export const useOpenings = ({ groupId: group_eq, type, openingsPositionType }: U
     () => data?.workingGroupOpenings.map((opening) => asWorkingGroupOpening(opening)) ?? [],
     [loading, data]
   )
+
+  const sortedOpenings= openings.slice().sort((a, b) => a.workingGroupOpening.updatedAt - b.workingGroupOpening.updatedAt)
```

Comment:
This needs to happen on the [WorkingGroupsOpenings](https://github.com/Joystream/pioneer/blob/dev/packages/ui/src/app/pages/WorkingGroups/WorkingGroupsOpenings.tsx) page.  Also note the failing test:
```
ERROR in src/working-groups/hooks/useOpenings.ts:46:59
16:31:06.075 | \| TS2339: Property 'workingGroupOpening' does not exist on type 'WorkingGroupOpening'.
16:31:06.075 | \|     44 \|   )
16:31:06.075 | \|     45 \|
16:31:06.075 | \|   > 46 \|   const sortedOpenings= openings.slice().sort((a, b) => a.workingGroupOpening.updatedAt -
```